modules/@apostrophecms/import-export-doc-type/index.js

```diff
@@ -8,7 +8,8 @@ module.exports = {
       action: 'export',
```

Comment:
Naming concern: this doesn't sound like a "criteria" object which is usually a mongodb query or something like one.